tests/main/interfaces-mount-control/task.yaml

```diff
@@ -36,30 +41,31 @@ execute: |
         snap connect "${SNAP_NAME}":mntctl
 
         echo "Verify that the snap can perform a mount"
-        mkdir -p "$MOUNT_DEST"
-        "${SNAP_NAME}".cmd mount -o bind,rw "$MOUNT_SRC" "$MOUNT_DEST"
+        mkdir -p "$MOUNT_DEST1"
+        "${SNAP_NAME}".cmd mount -o bind,rw "$MOUNT_SRC" "$MOUNT_DEST1"
 
         echo "Verify that the mount has been performed"
-        "${SNAP_NAME}".cmd grep "$MOUNT_DEST" /proc/self/mountinfo
+        "${SNAP_NAME}".cmd grep "$MOUNT_DEST1" /proc/self/mountinfo
 
         echo "and that it's only in the snap's namespace"
-        NOMATCH "$MOUNT_DEST" < /proc/self/mountinfo
+        NOMATCH "$MOUNT_DEST1" < /proc/self/mountinfo
 
         echo "Ensure that the mounted files are visible"
-        "${SNAP_NAME}".cmd test -e "$MOUNT_DEST/file1"
+        "${SNAP_NAME}".cmd test -e "$MOUNT_DEST1/file1"
 
         echo "Unmount via the system command umount(8)"
-        "${SNAP_NAME}".cmd umount "$MOUNT_DEST"
-        if "${SNAP_NAME}".cmd grep "$MOUNT_DEST" /proc/self/mountinfo; then
+        "${SNAP_NAME}".cmd umount "$MOUNT_DEST1"
+        if "${SNAP_NAME}".cmd grep "$MOUNT_DEST1" /proc/self/mountinfo; then
             echo "Unmount failed"
             exit 1
         fi
-        "${SNAP_NAME}".cmd test "!" -e "$MOUNT_DEST/file1"
+        "${SNAP_NAME}".cmd test "!" -e "$MOUNT_DEST1/file1"
 
         echo "Verify that a mount with a specific FS type can be created"
-        "${SNAP_NAME}".cmd mount -o rw -t tmpfs none "$MOUNT_DEST"
-        "${SNAP_NAME}".cmd grep "$MOUNT_DEST.*tmpfs" /proc/self/mountinfo
-        "${SNAP_NAME}".cmd umount "$MOUNT_DEST"
+        "${SNAP_NAME}".cmd mkdir -p "$MOUNT_DEST2"
+        "${SNAP_NAME}".cmd mount -o rw -t tmpfs none "$MOUNT_DEST2/"
+        "${SNAP_NAME}".cmd grep "$MOUNT_DEST2.*tmpfs" /proc/self/mountinfo
+        "${SNAP_NAME}".cmd umount "$MOUNT_DEST2/"
```

Comment:
(nipitck^9) in cases like this I kinda prefer `${MOUNT_DEST2}/` as it makes the boundary between var and "/" more clear but it does not matter.